web_src/js/components/ContextPopup.vue

```diff
@@ -108,7 +102,7 @@ export default {
       <p><small>{{ issue.repository.full_name }} on {{ createdAt }}</small></p>
       <p><svg-icon :name="icon" :class="['text', color]"/> <strong>{{ issue.title }}</strong> #{{ issue.number }}</p>
       <p>{{ body }}</p>
-      <div>
+      <div class="tw-flex tw-flex-wrap tw-gap-1">
```

Comment:
I think it could be `flex-text-block tw-flex-wrap`

Reply:
Even better, we can share `#issue-list .flex-item-title .labels-list` style.

Reply:
Done in https://github.com/go-gitea/gitea/pull/30237/commits/a26de2a6752ff117b8121f8fc1cd359b698e6691, now there is a single shared `labels-list` class.